resources/views/profiles/show.blade.php

```diff
@@ -38,6 +38,7 @@
 						@if($information->title) <h6>{{ $information->title }}</h6> @endif
 						@if($information->secondary_title) <h6>{{ $information->secondary_title }}</h6> @endif
 						@if($information->tertiary_title) <h6>{{ $information->tertiary_title }}</h6> @endif
+						@if($information->profile_summary) <p class="profile_summary">{{ $information->profile_summary }}</p> @endif
```

Comment:
Looks good, but maybe add the `mt-sm-0` class to the h2 above it to help balance out the spacing when they have a lot of items.